src/Snowflake/Command.php

```diff
@@ -125,12 +163,29 @@ public static function cloneDatabaseFromShared(Connection $connection, array $da
                 }
                 $schemaName = $schema['name'];
 
+                $schemaGrants = array_filter(
+                    $schemasGrants,
+                    function (array $v) use ($database, $schemaName) {
+                        $validSchema = [
+                            sprintf('%s.%s', $database, $schemaName),
+                            sprintf('%s.%s', $database, QueryBuilder::quoteIdentifier($schemaName)),
+                        ];
+                        return in_array($v['name'], $validSchema);
+                    }
+                );
+                $ownershipOnSchema = current(array_filter($schemaGrants, fn($v) => $v['privilege'] === 'OWNERSHIP'));
```

Comment:
bylo by lepší to mít bezpečně na dva kroky a mít tam assert, že výsledekem toho filtru je skutečně jen jeden řádek

Reply:
https://github.com/keboola/project-migration-tool/pull/1/commits/e3ef4bd0b54b6a1d6e78be2fa2c383df436f6657